### PR TITLE
Update footer and feedback

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
This PR is to update to the new footer and feedback design and also make some content changes

### How to review 
Check the footer and feedback is now the new design (some links may still not work as the pages for them are not live yet.

### CODE REVIEW

File: templates/layouts/configs/_feedback.html [hunk 1]
```diff
@@ -1,57 +1,21 @@
...
+            "value": page.title
+        },
+        "pageURL": {
+            "value": page.url
+        },
+        "mailto": {
+            "address": "census2021.feedback@ons.gov.uk",
+            "textBefore": _("Tell us what you think of this website by emailing"),
+            "textAfter": _("This email is for feedback only and we will not be able to reply, do not include any personal details."),
+            "subject": _("Census%202021%20feedback%3A%20"),
```
Comment (MebinAbraham) [inline templates/layouts/configs/_feedback.html]:
```jinja
            "subject": _("Census 2021 feedback: "),
            "body": ("\n\n\n---\n" ~ _("This email is for feedback on the census rehearsal, which will help us to improve Census 2021. We are unable to respond to these emails.\nIf you need help with your census, visit https://www.census.gov.uk/contact-us.")) | urlencode,
```

File: templates/layouts/configs/_feedback.html [hunk 2]
```diff
@@ -1,57 +1,21 @@
...
+            "body": _("%0D%0A%0D%0A%0D%0A%2D%2D%2D%0D%0AThis%20email%20is%20for%20feedback%20on%20the%20census%20rehearsal%2C%20which%20will%20help%20us%20to%20improve%20Census%202021.%20We%20are%20unable%20to%20respond%20to%20these%20emails.%0D%0AIf%20you%20need%20help%20with%20your%20census%2C%20visit%20https://www.census.gov.uk%2Fcontact-us.)"
```
Comment (MebinAbraham) [inline templates/layouts/configs/_feedback.html]:
```jinja
            "subject": _("Census 2021 feedback: "),
            "body": ("\n\n\n---\n" ~ _("This email is for feedback on the census rehearsal, which will help us to improve Census 2021. We are unable to respond to these emails.\nIf you need help with your census, visit https://www.census.gov.uk/contact-us.")) | urlencode,
```

**Review by MebinAbraham — DISMISSED** [review]
Has the colours been changed in the design system?. This is on click.
Screenshot 2019-09-18 at 18 51 07

**Comment by rmccar** [comment]
> Has the colours been changed in the design system?. This is on click.
> Screenshot 2019-09-18 at 18 51 07

Yeah that was part of a PR to address an issue pulled up in the DAC report. It's to do with the contrast on focus not being enough before.

https://github.com/ONSdigital/design-system/pull/593

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
I get an extra twisty character above the feedback link:
image

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
When I clicked the mailto link it opened gmail in the same tab, it should probably match the other links in the footer and open a new tab.

File: templates/layouts/_base.html [hunk 3]
```diff
@@ -49,11 +49,14 @@
 
     <!-- Google Tag Manager -->
     <script nonce="{{ csp_nonce() }}">
+      var a = /^(.*)?\s*'usage':true\s*[^;]+(.*)?$/;
```
Comment (MebinAbraham) [inline templates/layouts/_base.html]: Can we have a more meaningful variable name and add a comment about what this does?

**Review by MebinAbraham — CHANGES_REQUESTED** [review]

File: templates/layouts/configs/_feedback.html [hunk 4]
```diff
@@ -1,57 +1,21 @@
...
-      "data-qa": "btn-feedback-submit",
-      "data-ga": "click",
-      "data-ga-category": "Feedback",
-      "data-ga-action": "Send"
-    }
-  },
-  "thankYouMessage": _("Thanks for your feedback"),
-  "errorMessage": _("There was an error sending your feedback.\n(Error code: {x})")
+   "id": "feedback",
+        "title": _("Give us feedback and help us improve this service"),
```
Comment (iwootten) [inline templates/layouts/configs/_feedback.html]: The indentation in this file is a bit wonky.

File: templates/layouts/_base.html [hunk 5]
```diff
@@ -49,11 +49,15 @@
 
     <!-- Google Tag Manager -->
     <script nonce="{{ csp_nonce() }}">
+      // depending on the cookie settings this if will allow the gtm snippet to load or stop it from loading
+      var gtmCookieSettings = /^(.*)?\s*'usage':true\s*[^;]+(.*)?$/;
+      if (document.cookie.match(gtmCookieSettings)) {
```
Comment (ajmaddaford) [inline templates/layouts/_base.html]: We shouldn't need to do this. The cookie could be read by flask and the `google_tag_manager_*` template variables not set if disabled.

File: templates/static/accessibility.html [hunk 6]
```diff
@@ -4,7 +4,21 @@
...
-  <h1>{{ _("Census questionnaire accessibility statement") }}</h1>
+  {% if schema_theme == "census-nisra" %}
+
+  {% set contact_us_url = "https://census.gov.uk/ni/contact-us/" %}
+
+{% elif language_code == "cy" %}
+
+  {% set contact_us_url = "https://cyfrifiad.gov.uk/cysylltu-a-ni/" %}
+
+{% else %}
```
Comment (iwootten) [inline templates/static/accessibility.html]: The indentation on these if statements is incorrect. Can't we set the url outside the if to the default and drop one of the switches?

Wouldn't it make more sense to set this url in the backend?

File: tests/functional/spec/question_definitions.spec.js [hunk 7]
```diff
@@ -11,69 +11,69 @@ describe('Component: Definition', function() {
...
+        .isVisible(DefinitionPage.definitionButton2()).should.eventually.be.false
 
         // When
-        .click(DefinitionPage.definitionTitle1())
+        .click(DefinitionPage.definitionTitle2())
 
-        // Then
-        .waitForVisible(DefinitionPage.definitionContent1(), 300)
-        .waitForVisible(DefinitionPage.definitionButton1(), 300);
+        // // Then
```
Comment (MebinAbraham) [inline tests/functional/spec/question_definitions.spec.js]:
```suggestion
        // Then
```

**Review by MebinAbraham — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

File: tests/integration/test_application_variables.py [hunk 8]
```diff
@@ -28,6 +28,10 @@ def test_flask_toolbar_is_displayed(self):
 
     def test_google_analytics_code_is_present(self):
         self.launchSurvey('test_textfield')
+        self._client.set_cookie(
```
Comment (MebinAbraham) [inline tests/integration/test_application_variables.py:31]: Need to rework in future to not use the private method.

**Pull request merged**